spec/factories/factories.rb

```diff
@@ -1,11 +1,32 @@
 FactoryGirl.define do
+  factory :category do
+    name
+    slug
+  end
```

Comment:
Do we need a slug for category? We are going to be slugging the  Charity name itself for vanity URLs instead of categories like we initially had. Let's change this to just name.